data/boarding missions.txt

```diff
@@ -37,3 +37,34 @@ mission "Assisting Hai"
 			`Knowing you risked your life to save the <origin>, the captain gives you <payment>.`
 				decline
 	destination Hai-home
+
+
+mission "Assisting Navy"
+	assisting
+	repeat
+	to offer
+		random < 27
+	source
+		government Republic
+	on offer
+		payment 25000
+		conversation
+			`The captain of the <origin> is greatful for your service to The Navy and gives you <payment>.`
+				decline
+	destination Farpoint
+
+
+mission "Assisting Militia"
+	assisting
+	repeat
+	to offer
+		random < 27
+	source
+		government Escort
```

Comment:
So you get paid for repeatedly disabling your own ships and then fixing them? Do you perhaps mean `government "Free Worlds"`?

Reply:
Or `government Militia`